ocp/gcp/05_implement_ocp_compliance_operator/implement_ocp_compliance_operator.md

```diff
@@ -137,38 +147,35 @@ An extract of the oc get ccr command can be seen below.
-ocp4-cis-api-server-basic-auth                                                 PASS     medium
-ocp4-cis-api-server-bind-address                                               PASS     low
-ocp4-cis-api-server-client-ca                                                  PASS     medium
-ocp4-cis-api-server-encryption-provider-cipher                                 FAIL     medium
-ocp4-cis-api-server-encryption-provider-config                                 FAIL     medium
-```
-
-Initially a number of the compliance checks will FAIL, many of them will be remediated automatically by the compliance operator. This remediation process can take sometime (hours). These remediation a actioned by the OCP cluster operators, using the following command the progress of these cluster operators can be seen.
+NAME                                                                     STATUS   SEVERITY
+ocp4-cis-scc-limit-container-allowed-capabilities                         PASS     medium
+ocp4-cis-scc-limit-ipc-namespace                                          MANUAL   medium
+ocp4-cis-scc-limit-net-raw-capability                                     MANUAL   medium
+ocp4-cis-scc-limit-network-namespace                                      MANUAL   medium
+ocp4-cis-scc-limit-privilege-escalation                                   MANUAL   medium
+ocp4-cis-scc-limit-privileged-containers                                  MANUAL   medium
+ocp4-cis-scc-limit-process-id-namespace                                   MANUAL   medium
+ocp4-cis-scc-limit-root-containers                                        MANUAL   medium
+ocp4-cis-secrets-consider-external-storage                                MANUAL   medium
+ocp4-cis-secrets-no-environment-variables                                 MANUAL   medium
+ocp4-pci-dss-accounts-restrict-service-account-tokens                     MANUAL   medium
+ocp4-pci-dss-accounts-unique-service-account                              MANUAL   medium
+ocp4-pci-dss-api-server-admission-control-plugin-alwaysadmit              PASS     medium
+ocp4-pci-dss-api-server-admission-control-plugin-alwayspullimages         PASS     high
+ocp4-pci-dss-api-server-admission-control-plugin-namespacelifecycle       PASS     medium
+ocp4-pci-dss-api-server-admission-control-plugin-noderestriction          PASS     medium
+ocp4-pci-dss-api-server-admission-control-plugin-scc                      PASS     medium
+```
+
+Initially a number of the compliance checks will FAIL, many of them will be remediated automatically by the compliance operator. This remediation process can take sometime (hours). These remediations are actioned by the OCP cluster operators, using the following command the progress of these cluster operators can be seen.
 
 '''shell
 oc get co
 ```
 Below is an example of the output of this command.
 
+#todo - update with new output when update to 4.13 takes place
```

Comment:
is this TODO intended to stay in the PR? up to you, just double checking.

Reply:
Yes. thanks for highlighting.